docs/src/user/manual/about.md

```diff
@@ -0,0 +1,37 @@
+---
+title: BIMS Documentation
+summary: Central documentation site for BIMS projects
+    - Tim Sutton
+    - Jeremy Prior
+    - Ketan Bamniya
+date: 26-09-2023
+some_url: https://github.com/kartoza/bims-website
+copyright: Copyright 2023, Kartoza
+contact: 
+license: This program is free software; you can redistribute it and/or modify it under the terms of the GNU Affero General Public License as published by the Free Software Foundation; either version 3 of the License, or (at your option) any later version.
+#context_id: 1234
+---
+
+# About
+
+This `About` section provides detailed information about the BIMS project. It offers insights into the project's objectives, purpose, and significance, helping users understand its scope and how it aims to address specific challenges. 
+
+[![Home Page](./img/about-img-1.png)](./img/about-img-1.png)
```

Comment:
Why are all the images relative links in all the files?

Standard images in markdown would be: ``![Image Name](path/to/image.png)``

Reply:
@Jeremy-Prior, once Eli told me to do that to show the image content properly.

Reply:
@Jeremy-Prior, once Eli told me to do that to show the image content properly.

Reply:
@ketankartoza I'm going to need more an explanation than that. Making images into links will not affect how the image displays on the site?

Reply:
@Jeremy-Prior, when we click on the image it will open that image in a new window in bigger size.

Reply:
That seems unnecessary, but sure, leave it then. Just address the video embedding.

Reply:
@Jeremy-Prior 
I am trying on my local but it's not working. Will it work on the live site only?

Reply:
@ketankartoza it should work when you do a local build, look at the Mobile application videos in the user guide.

Reply:
@Jeremy-Prior 

![image](https://github.com/user-attachments/assets/608713c7-3bb6-418f-adc0-f8073c302742)

Reply:
Yes, it will look like that in VSCode, do a ./build-docs-html.sh and mkdocs serve to view the site on your local.

Reply:
It should look like this when you do a `mkdocs serve` from your end:

![image](https://github.com/user-attachments/assets/7f5b386f-081e-4774-96fa-0fff6686a79a)